psi4/src/psi4/libsapt_solver/fdds_disp.cc

```diff
@@ -155,23 +156,83 @@ FDDS_Dispersion::FDDS_Dispersion(std::shared_ptr<BasisSet> primary, std::shared_
     // Build DFHelper
     dfh_ = std::make_shared<DFHelper>(primary_, auxiliary_);
     dfh_->set_memory(doubles);
-    dfh_->set_method("DIRECT_iaQ");
+    if (is_hybrid) {
```

Comment:
```suggestion
    if (is_hybrid_) {
```
this is maybe a copy/paste error from yesterday?

Reply:
Looks like so. Should I click "commit suggestion" here? If so, how would I synchronize it locally?